application/src/ext-test/java/org/opentripplanner/ext/flex/FlexStopTimesForTest.java

```diff
@@ -27,26 +29,74 @@ public static StopTime area(StopLocation areaStop, String endTime, String startT
     stopTime.setStop(areaStop);
     stopTime.setFlexWindowStart(TimeUtils.time(startTime));
     stopTime.setFlexWindowEnd(TimeUtils.time(endTime));
+    stopTime.setTrip(TRIP);
     return stopTime;
   }
 
-  public static StopTime regularArrival(String arrivalTime) {
-    return regularStopTime(TimeUtils.time(arrivalTime), MISSING_VALUE);
+  /**
+   * Returns an invalid combination of a flex area and continuous stopping.
+   */
+  public static StopTime areaWithContinuousStopping(String time) {
+    var st = area(time, time);
+    st.setFlexContinuousPickup(PickDrop.COORDINATE_WITH_DRIVER);
+    st.setFlexContinuousDropOff(PickDrop.COORDINATE_WITH_DRIVER);
```

Comment:
Does continuous drop-off imply continuous pick-up? (That also makes this test case overlap with the next one.)

Reply:
In the spec these are completely different things but of course you would be hard pressed to find an example in the real world where you can only be dropped off continuously but not picked up.